src/Qt/PlayerPrivate.cpp

```diff
@@ -55,12 +56,16 @@ namespace openshot
 
         using std::chrono::duration_cast;
 
-        // Types for storing time durations in whole and fractional milliseconds
+        // Types for storing time durations in whole and fractional microseconds
         using micro_sec = std::chrono::microseconds;
         using double_micro_sec = std::chrono::duration<double, micro_sec::period>;
 
         while (!threadShouldExit()) {
-            // Calculate on-screen time for a single frame in milliseconds
+            if (skipFrames > 0) {
```

Comment:
I don't think this actually does anything. It just quickly subtracts whatever you put in this variable back to 0 (almost instantaneously). It does not call `getFrame()` or increment the video position.

Reply:
Updated, so it should be updating the frame number, as a way to jump ahead.